src/ray/gcs/gcs_client/gcs_client.cc

```diff
@@ -147,7 +147,8 @@ PythonGcsClient::PythonGcsClient(const GcsClientOptions &options) : options_(opt
 
 Status PythonGcsClient::Connect() {
   auto arguments = PythonGrpcChannelArguments();
```

Comment:
`arguments` can also be removed now and problably so can the whole `PythonGrpcChannelArguments` function :)